tests/run-make/incr-prev-body-beyond-eof/Makefile

```diff
@@ -1,19 +0,0 @@
-# ignore-none no-std is not supported
```

Comment:
Question: This confuses me greatly, do xxx-xxx-none target triples correspond to no-std?

Reply:
My uninformed guess is that the test fails on a minimalist embedded system with no "real" operating system or standard library (naturally). But, is that important? This test will probably never run on such a device.